clang/include/clang/Basic/Builtins.td

```diff
@@ -896,6 +896,12 @@ def Launder : Builtin {
   let Prototype = "void*(void*)";
 }
 
+def StartObjectLifeTime : Builtin {
```

Comment:
Nit: lifetime is one word not two, so the `T` should not be capitalized.

Reply:
Done.